ansible/files/multinode.sh

```diff
@@ -368,6 +368,12 @@ function upgrade_overcloud() {
   run_kayobe overcloud service upgrade
 }
 
+function upgrade_prerequisites() {
+  # Run the upgrade prerequisites script if it exists.
+  workaround_ansible_rc13_bug
+  [[ ! -f $KAYOBE_CONFIG_PATH/../../tools/upgrade-prerequisites.sh ]] || $KAYOBE_CONFIG_PATH/../../tools/upgrade-prerequisites.sh
```

Comment:
Can you use $KAYOBE_CONFIG_ROOT here, instead of $KAYOBE_CONFIG_PATH/../.. ?

Reply:
I don't think that's a standard variable. I just checked a prod system and it's not defined

Reply:
It's set when you source the [kayobe-env](https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/2024.1/kayobe-env#L15) script, but not exporter, so you won't see it in `env | grep KAYOBE` for example. Should still be able to use it.

Reply:
Interesting. Is it definitely within scope?